src/__tests__/sheet-test.tsx

```diff
@@ -43,7 +47,7 @@ test('Sheet', async () => {
 
     const closeButton = await within(sheet).findByRole('button', {name: 'Close'});
 
-    await act(() => userEvent.click(closeButton));
+    await act(async () => userEvent.click(closeButton));
```

Comment:
I know it was there before this PR, but the act() shouldn't be needed. useEvent.click() already uses it in its internal implementation. Maybe you can remove them

Reply:
If I remove this act, the test won't pass. I get a timeout in the next assertion, even if I increase the timeout time

Reply:
Discussed offline. It seems that there is something weird happening with waitForElementToBeRemoved(). Let's change this to something like:

![MicrosoftTeams-image](https://github.com/user-attachments/assets/c46541bc-e7b9-41b5-ae96-6ad40d1fce20)

Reply:
as talked. The act was removed and the next expectation just checks that the element does not exist in the dom